### ISSUE TITLE
React component: SVG icon

### ISSUE THREAD

**Issue by sarahmonster** [a8cteam51/strikebase#83]
I'm *hoping* once we have a structure sorted out, I should be able to figure out how to implement an SVG icon system using React.

https://css-tricks.com/creating-svg-icon-system-react/
https://dmfrancisco.github.io/react-icons/
https://medium.com/@david.gilbertson/icons-as-react-components-de3e33cb8792#.6mh9n1cxn

**Comment by sarahmonster** [comment]
@danielwrobert Is this something that I can or should implement at this stage, or is it best to move this as a v3 task?

**Comment by danielwrobert** [comment]
I think the advantage of moving it to a v3 task is that we can discuss the approach to make sure we're on the same page with things, such as how to best structure the components to fit in our application. I can also walk you through everything on the React branch in person at our meetup, if you wanted.

That said I also don't think it'll hurt anything to get started in a separate branch/PR if you want to. It's not like we can't get communication going there just as well. These components will just be presentational so there's no concern with merging in functionality or anything.

**Comment by sarahmonster** [comment]
I'm going to make the executive decision🕴🏾to move to v3, since I think that'll make it a bit simpler to work on. Once we've closed all the v2 issues this will be first on my List.

**Comment by danielwrobert** [comment]
Good call, I agree with this. I just didn't want to prevent you from tinkering with it if you really wanted to, as it wouldn't have a huge impact either way. :)
